buildozer/__init__.py

```diff
@@ -361,22 +362,22 @@ def check_build_layout(self):
             exit(1)
 
         # create global dir
-        self.mkdir(self.global_buildozer_dir)
-        self.mkdir(self.global_cache_dir)
+        mkdir(self.global_buildozer_dir)
```

Comment:
To help people understand that `mkdir` (and other methods) are from `buildops`, how about importing `buildops` and than using `buildops.mkdir`?

Yeah, is quite verbose, but IMHO, the readability (and the understandability) gets improved a lot.

What do you think about it?